cmd/syncthing/main.go

```diff
@@ -763,7 +763,7 @@ func resetDB() error {
 
 func standbyMonitor(app *syncthing.App, cfg config.Wrapper) {
 	restartDelay := 60 * time.Second
-	now := time.Now()
+	now := time.Now().Round(0)
```

Comment:
Could you please add a comment explaining that rounding ensures wall time is used. Or make a tiny `wallTimeNow` utility method, such that the name is hte explanation.